installer/Templates/Permissions.php

```diff
@@ -0,0 +1,37 @@
+<?php /** @noinspection PhpUnused */
+
+
+namespace Installer\Templates;
+
+use Template;
+
+class Permissions implements Template
+{
+
+	public function print(array $input = [])
+	{
+		echo "\t".'<ul class="list">'."\n";
+		foreach ($input['permissions'] as $permission) {
+			echo "\t"."\t".'<li class="list__item list__item--permissions '
+				.($permission['isSet'] ? 'success' : 'error').'">'."\n";
+			echo "\t"."\t".$permission['folder']."\n";
```

Comment:
I'm having difficulty finding it in the source code (it must be somewhere in the CSS, I guess) but I don't like the fact that the pathnames displayed in the web browser start with an uppercase...

Reply:
I did not look into it.